Formula/p/python@3.12.rb

```diff
@@ -394,13 +394,25 @@ def post_install
        xyz, where xyz is the package you are trying to
        install.
 
-       If you wish to install a non-brew-packaged Python package,
-       create a virtual environment using python3 -m venv path/to/venv.
-       Then use path/to/venv/bin/python and path/to/venv/bin/pip.
+       If you wish to install a Python library that isn't in Homebrew,
+       use a virtual environment:
 
-       If you wish to install a non-brew packaged Python application,
-       it may be easiest to use pipx install xyz, which will manage a
-       virtual environment for you. Make sure you have pipx installed.
+         python -m venv path/to/venv
+         source env/bin/activate
+         python -m pip install xyz
+
+       If you wish to install a Python application that isn't in Homebrew,
+       it may be easiest to use 'pipx install xyz', which will manage a
+       virtual environment for you. You can install pipx with
+
+         brew install pipx
+
+       Expert users may restore the old behavior of pip by passing
+       the '--break-system-packages' flag to pip, or by adding
+       'break-system-packages = true' to their pip.conf file. The latter
+       will permanently disable this warning.
```

Comment:
Maybe worth mentioning that it's strongly recommended to also pass/set `--user` to avoid the risk of `brew link` errors if you happen to install something brew ships.

Reply:
Yeah, good catch -- I think this should only mention the `--user` case, since any other case is likely to break Homebrew + we strongly want to discourage anyways. I'll update today!

Reply:
Updated, PTAL! Also fixed the use of second person to be consistent with the rest of the error.